### ISSUE TITLE
Support sending emails using integration's own templates

### ISSUE THREAD

**Issue by pmgarman** [affiliatewp/AffiliateWP#963]
There is an option in the Emails settings to choose a template, you can choose from the default template or plain text emails. Having the ability to choose to send emails through the email template of one of the enabled integrations, only those that both include custom email templates AND a way to send emails through them of course, would be a huge bonus.

This example is specific to WooCommerce but for any integration that supports sending custom emails through its own templates this is all relevent. We've just signifigantly customized the WooCommerce transactional email templates. In doing so these emails and other emails from the site that we send through here are all going to be styled and branded a certain way. It would be *awesome* if AffiliateWP could piggy back off those customizations by including some method of sending those emails through the templates of WooCommerce instead of it's own by choosing WooCommerce from that dropdown box when WooCommerce is enabled. Keeping the affiliate emails of the WooCommerce store styled and branded the same as all other store emails.

**Comment by michaelbeil** [comment]
I like that idea @pmgarman.  I wonder if we could just search for the template to see if it is available and list it in the drop-down.

Thoughts @pippinsplugins and @sumobi?

**Comment by pippinsplugins** [comment]
Oh man, that's such a good idea. Love love love.

**Comment by pmgarman** [comment]
:D @pippinsplugins if you guys get the AffWP side of things created to support it, I'm happy to jump in on the WooCommerce implementation to take your email content and wrap it in the WooCommerce template.

**Comment by pippinsplugins** [comment]
:+1:

**Comment by sumobi** [comment]
Neat idea :+1:

**Comment by VictorSlate** [comment]
Good idea! Actually I was trying to find a solution and found this topic.
@pippinsplugins Do you planning to realize this?

**Comment by pippinsplugins** [comment]
I would definitely like to make it happen but there's no timeframe at this point.
